Renumbering.py

```diff
@@ -5,16 +5,16 @@
 input = open("missing_linkers_final.txt", 'r')
 i = 0
 for line in input:
-    l = line.split()
-    if len(l) == 8:
+    L = line.split()
+    if len(L) == 8:
```

Comment:
I think you have this backwards... L should be l, not the other way! Let me know if you have any questions about this...